content/guides/integrations/memcached/Python.md

```diff
@@ -0,0 +1,248 @@
+```
+
+### Build it
+Change directory to the newly forked pymemcache repository and in there run
+```shell
+python setup.py install
+```
+
+and then use it normally like you would have used the original package!
+
+However, for a guided example,  please continue reading below.
+
+## Problem statement
+
+Our sample is an application excerpt from a distributed prime factorization engine that needs to calculate square roots of big numbers but would
+like to reuse expensively calculated results since square roots of such numbers are CPU intensive.
+
+To share/memoize results amongst our distributed applications, we’ll use Memcache.
+However, to examine our changes in a distributed systems deployment with numerous microservices, it is useful to have tracing to verify our optimizations
+and inspect the state of the system.
+
+This is what the code original looks like
+
+{{% notice tip %}}
+For assistance installing Memcached, please visit the [Memcached Installation wiki](https://github.com/memcached/memcached/wiki/Install)
+{{% /notice %}}
+
+Please run your Memcached server, for this example we'll run it locally on port 11211.
+
+{{<highlight python>}}
+#!/usr/bin/env python
+
+import math
+import os
+
+from pymemcache.client.base import Client as MemcacheClient
+
+def numAsStr(num): return '%f'%(num)
+
+def main():
+    mc = MemcacheClient(("localhost", 11211,))
+
+    values = [
+        9999999183838328567567689828757567669797060958585737272727311111199911188888889953113,
+        9082782799999992727727272727272727272727272727838383285677272727399917272728373799422229,
+        987773333733799999999475786161616373683838328567727272739991727272837378888888881322229,
+        99999999999999999999998899919191919191919192828727737368383832856772727273999172727283737671626262269222848,
+        991818178171928287277373683838328567727272739991727272837377827272727272727272727711119,
+    ]
+
+    for value in values:
+        for i in range(0, 2):
+            sqrt = calculateSQRT(mc, value)
+            print("%-100.10f ==> %-100.10f"%(value, sqrt))
+
+        # Afterwards, clean up to ensure repeatability of the test
+        mc.delete(numAsStr(value))
+
+def calculateSQRT(mc, num):
+    # Firstly check if we've cached it
+    numStr = numAsStr(num)
+    res = mc.get(numStr)
+    if res is not None:
+        try: # Try parsing it as a float
+            return float(res)
+        except ValueError:
+            # Failed to parse it
+            _ = 0
+
+    # Otherwise this was a cache miss, so go on
+    sqrt = math.sqrt(num)
+
+    sqrtStr = numAsStr(sqrt)
+    # Now cache it for a cache hit later on
+    mc.set(numStr, sqrtStr)
+    return sqrt
+
+if __name__ == '__main__':
+    main()
+{{</highlight>}}
+
+## Enabling OpenCensus
+To provide observability, we'll enable tracing by importing OpenCensus Python's tracing package
+
+{{<highlight python>}}
+from opencensus.trace.tracer import Tracer
+
+# For demo purposes, we'll always sample
```

Comment:
nit: I think this is a h3 header `###`

Reply:
Oh, this is part of the source code :)
<img width="1037" alt="screen shot 2018-08-27 at 11 14 10 am" src="https://user-images.githubusercontent.com/4898263/44674303-59de9180-a9ea-11e8-8b72-72bf0c1bc26d.png">